contracts/fio.system/src/voting.cpp

```diff
@@ -1225,12 +1225,16 @@ namespace eosiosystem {
             //check(!isproxy || !pitr->proxy, "account that uses a proxy is not allowed to become a proxy");
             if (isproxy && !pitr->proxy) {
                 votersbyowner.modify(pitr, same_payer, [&](auto &p) {
+                    p.fioaddress = fio_address;
```

Comment:
Not needed. See Issue Notes.

Reply:
Please check my reply in the issue. This is a workaround for the problem that happened with our proxy. Probably there's a better way to implement a definitive solution, but this solves the problem in the mid term.